src/_includes/layouts/post.njk

```diff
@@ -35,40 +36,24 @@
                 <div class="prose">
                     <div class="mb-4 hero-img">
                         {% if video %}
-                        <iframe width="560" height="315" src="https://www.youtube.com/embed/{{video}}"
+                            <iframe width="560" height="315" src="https://www.youtube.com/embed/{{video}}"
                             title="YouTube video player" frameborder="0"
                             allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
                             allowfullscreen></iframe>
                         {% elif image %}
-                        <div class="w-full h-auto">
-                            {% set imageSrc = ["./", image ] | join %}
-                            {% set imageDescription = ["Image representing ", title] | join %}
-                            {% image imageSrc, imageDescription, [560] %}
-                        </div>
+                            <div class="w-full h-auto">
+                                {% set imageSrc = ["./", image] | join %}
+                                {% set imageDescription = ["Image representing ", title] | join %}
+                                {% image imageSrc,
+                                imageDescription,
+                                [560] %}
+                            </div>
                         {% else %}
-                        {{ page.url | generatePostSVG | safe }}
+                            {{ page.url | generatePostSVG | safe }}
                         {% endif %}
                     </div>
                     {{ content | safe }}
                 </div>
-                <div class="mt-9 pt-3 border-t-2">
-                    <h3 class="mb-3">Comments:</h3>
-                    <div id='discourse-comments'></div>
-                </div>
-                <script type="text/javascript">
-                    window.DiscourseEmbed = {
-                        discourseUrl: 'https://community.flowfuse.com/',
-                        discourseEmbedUrl: "{{ site.baseURL }}{{ page.url }}",
-                        className: 'ff-comment-thread'
-                    };
-                    (function () {
-                        var d = document.createElement('script');
-                        d.type = 'text/javascript';
-                        d.async = true;
-                        d.src = window.DiscourseEmbed.discourseUrl + 'javascripts/embed.js';
-                        (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(d);
-                    })();
-                </script>
```

Comment:
This is the real change, the rest is reformatting the doc. Sorry for the noisy diff.

Reply:
I'm seeing this appears now below the footer, but I'm not sure why... 

<img width="1527" alt="Screenshot 2024-01-15 at 14 13 10" src="https://github.com/FlowFuse/website/assets/129537638/d7fb7a83-fe57-44b9-9a6f-54d30b21da59">
 

And the sign up form disappeared

<img width="311" alt="Screenshot 2024-01-15 at 14 49 15" src="https://github.com/FlowFuse/website/assets/129537638/a7747870-c695-415f-a7a9-550491bdd390">